neuron/src/app/Neuron/Config.hs

```diff
@@ -37,10 +39,10 @@ getConfig = do
   configPath <- ribInputDir <&> (</> configFile)
   configVal :: Text <-
     liftIO (doesFileExist configPath) >>= \case
-      True -> do
-        mergeWithDefault . toText <$> readFile' configPath
+      True ->
+        toText <$> readFile' configPath
       False ->
-        pure defaultConfig
+        pure emptyConfig
   either fail pure $ parsePure configFile $ mergeWithDefault configVal
```

Comment:
Hmm, what's the reason for this change? Default config is not used anymore?

Reply:
Ahh, the introduction of `emptyConfig` was more important in a previous iteration of the code. We can remove it.

With that said, the `Config` interface was a little strange before. Exporting both `defaultConfig` and `mergeWithDefault` felt like a bit of a code smell. And on the line after this one, we would sometimes end up merging two copies of `defaultConfig`.

Proposed solution: change the `Config` interface to export neither `emptyConfig` nor `defaultConfig`, but change `mergeWithDefault` to
```haskell
mergeWithDefault :: Maybe Text -> Text
mergeWithDefault = \case
  Nothing -> defaultConfig
  Just config -> defaultConfig <> " // " <> config
```
Then this line of code can be `pure Nothing`.

Reply:
Sounds good, re: proposed solution.

Reply:
No need to make this change in this PR (see comment below).